code-challenges.test.js

```diff
@@ -11,46 +11,176 @@
 // Run the file with the following command: $ yarn jest
 
 
-// --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
+// QUESTION 1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
 
-// a) Create a test with expect statements for each of the variables provided.
 
-const fibLength1 = 6
+//PSUEDO CODING
+// create a functionn that takes in a number > 2 (greater than 2)
+// Output: Returns an array the same length while addinig(.push) to push the new number into the empty array. Then add (+) the previous number to each to get the fibinochi sequence
+// Create variables to allow the fibinochi sequence to be pushed onto a new array
 // Expected output: [1, 1, 2, 3, 5, 8]
-
-const fibLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
 
-// b) Create the function that makes the test pass.
+  const fibLength1 = 6
+//  Expected output: [1, 1, 2, 3, 5, 8]
 
+ const fibLength2 = 10
+//  Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
-// --------------------2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+// FAIL  ./code-challenges.test.js
+// fibinochi
+//   ✕ returns an array that length containing the numbers of the Fibonacci sequence using a number > than 2 (4 ms)
+
+// ● fibinochi › returns an array that length containing the numbers of the Fibonacci sequence using a number > than 2
+
+//   expect(received).toEqual(expected) // deep equality
+
+//   Expected: [1, 1, 2, 3, 5, 8]
+//   Received: undefined
+
+describe("fibinochi", () => {
+    it("returns an array that length containing the numbers of the Fibonacci sequence using a number > than 2", () => {
+      expect(fibinochi(fibLength1)).toEqual([1, 1, 2, 3, 5, 8]);
+     expect(fibinochi(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55]);
+    });
+  })
+
+ 
+
+
+// b) Create the function that makes the test pass.
+
+// ONLY WAS ABLE TO GET ON TEST TO PASS. SPENT THE WHOL WEEKEND ON JUST THIS
+
+
+let fibinochi= () => {}
+    const newArray = [0, 1];
+    for(var i = 2; i < newArray; i++) {
+       const a = (i - 1);
+       const b = (i - 1);
+       newArray.push(a + b);
+    }
```

Comment:
This is a great attempt and honestly it has all the ingredients for the recipe here! But putting the ingredients together correctly is also half the journey. 

Go and lookup function syntax in javascript unit 1 and check out what's going on with the curly brackets or "code block"